portal/templates/login/index.njk

```diff
@@ -1,41 +1,51 @@
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/password-input/macro.njk" import govukPasswordInput %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% extends "index.njk" %}
+
 {% block pageTitle -%}
   Log in
 {%- endblock %}
+
 {% block content %}
-{% if errors %}
-  {{ govukErrorSummary({
+
+  {% if errors %}
+    {{ govukErrorSummary({
       titleText: "There is a problem",
       errorList: errors.errorSummary,
       attributes: {
-        'data-cy': 'error-summary'
+        "data-cy": "error-summary"
       }
     }) }}
-{% endif %}
-{% if passwordreset %}
-  {{ govukInsetText({
+  {% endif %}
+
+  {% if passwordreset %}
+    {{ govukInsetText({
       classes: "govuk-!-margin-top-0",
       html: '<p>We\'ve sent an email to the address you have provided. Please check your inbox and spam folder for a message from us. </p><p>If you require further assistance please contact <a class="govuk-link" href="mailTo:' + CONTACT_US_EMAIL_ADDRESS + '"> ' + CONTACT_US_EMAIL_ADDRESS + '</a>.</p>',
```

Comment:
beyond the scope maybe but would be good if this was a {% set passwordResetHtml .... instead

Reply:
Agree that'd be great to do outside of this PR.